### ISSUE TITLE
0.7.3.3 Law Crashes server instantly.

### ISSUE THREAD

**Issue by DeathTech154** [StrangeLoopGames/EcoIssues#7633]
Hey guys,
Here is another one.

While testing the law system I found in some cases or well. In most cases.

Changing a law where you have a comparison like < will crash the whole server when you change the < to > or =

Pretty reliably. Not sure if anyone complained about that before.
If it wasn't fixed in 7.3.3 it will most definitely still be in 7.4.

I mostly ran into this issue trying to make a limitation law for hunting / crafting of skillbooks / crafting of a mint.

**Comment by MarjieAVolk** [comment]
You mean this happens just while setting up the law proposal? :O

**Comment by MarjieAVolk** [comment]
Looks like you need to change it to > or = and then change it back to <, but yeah I'm seeing it.  O.o

**Comment by DeathTech154** [comment]
Yeah, the crash happens instantly if changing directly for some reason.
I noticed this behaviour several weeks ago but thought it was the server being unstable.

Until like the 5th time I finally noticed it always happens when changing "<" to ">".
The server never seemingly gave me a reason just locked up and crashed.

It's an issue that sadly can easily be abused to totally and consistently crash a server and people maybe don't even know they are causing it.
